### PR TITLE
Optimization of getTargetZoom () function

### PR DESCRIPTION
**Pull request by christophekede**
## PR Checklist

Please check all that apply to this PR using "x":

- [ x] I have checked that this PR is not a duplicate of an existing PR (open, closed or merged)
- [ ] I have checked that this PR does not introduce a breaking change
- [ x] This PR introduces breaking changes and I have provided a detailed explanation below

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting)
- [x] Refactoring (no functional changes)
- [ ] Documentation changes
- [ ] Other - Please describe:

## Fixes

Issue Number: 67

## What is the new behavior?
Optimization of getTargetZoom
## Other information
You have to change the port in package.json, i put 3000

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "description": "An open source and touch friendly web based comic book reader as a react component",
   "main": "./dist/villain.js",
   "scripts": {
-    "start": "webpack-dev-server --config ./webpack.config.js --mode development --open --hot",
+    "start": "webpack-dev-server --config ./webpack.config.js --mode development --open --hot  --port 3000",
```
Comment (btzr-io) [inline package.json]: I'm not sure about this change, is this really necessary ?

File: src/components/render.js [hunk 2]
```diff
@@ -24,19 +24,16 @@ class CanvasRender extends Component {
...
     const count = world.getItemCount()
     // MAX: Original size (1:1)
-    if (count > 1) {
-      const tile = world.getItemAt(0)
-      return tile.imageToViewportZoom(scale)
-    } else if (count && count === 1) {
-      return viewport.imageToViewportZoom(scale)
-    }
-  }
+    return count > 1 ?  world.getItemAt(0).imageToViewportZoom(scale) : viewport.imageToViewportZoom(scale)
```
Comment (btzr-io) [inline src/components/render.js]: how is this optimized care to explain ?
Also know is really hard to read / understand

File: package.json (hunk 1 above)
Comment (christophekede) [reply package.json]: this change was for me because default port 8080 was used by another process

Comment (btzr-io) [reply package.json]: Oh, ok, maybe you could open an issue about this to improve the development workflow, thanks :v: ^^

File: src/components/render.js (hunk 2 above)
Comment (christophekede) [reply src/components/render.js]: I removed one condition because  "if" "else" statement decreases program performance, and i used ternary expression

**Comment by btzr-io** [comment]
I dont think this changes have a real effect on performance, please provide more info.
I'll close this for now, feel free the check other more simple issues so you can learn more about the project, thanks :v:

**Pull request closed without merging**